azurerm/azurerm_linux_virtual_machine/simple/main.tf

```diff
@@ -0,0 +1,97 @@
+# Summary: A simple Azure Linux Virtual Machine
+
+# Documentation: https://www.terraform.io/docs/language/settings/index.html
+terraform {
+  required_version = ">= 0.14.0"
+  required_providers {
+    azurerm = {
+      source  = "hashicorp/azurerm"
+      version = "~> 2.0"
+    }
+  }
+}
+
+# Documentation: https://www.terraform.io/docs/language/values/variables.html
+variable "azure_subscription_id" {
+  type = string
+}
+
+# Documentation: https://www.terraform.io/docs/language/values/variables.html
+variable "changeme_ssh_public_key" {
+  type = string
+}
```

Comment:
This can't be applied automatically by CI.
We should have a default value for such variable (or define the `admin_ssh_key` in some different way).

Reply:
Fixed with: https://github.com/ContainerSolutions/terraform-examples/pull/62/commits/9172c50228db25b77c7df13c910daeb4affbb59a